src/components/views/messages/TextualBody.js

```diff
@@ -545,20 +548,23 @@ export default class TextualBody extends React.Component {
                         &nbsp;
                         { body }
                         { widgets }
+                        { this.props.scBubbleGroupTimestamp }
                     </span>
                 );
             case "m.notice":
                 return (
-                    <span className="mx_MNoticeBody mx_EventTile_content">
+                    <span className={ "mx_MNoticeBody mx_EventTile_content" } dir="auto">
                         { body }
                         { widgets }
+                        { this.props.scBubbleGroupTimestamp }
                     </span>
                 );
             default: // including "m.text"
                 return (
-                    <span className="mx_MTextBody mx_EventTile_content">
+                    <span className={ "mx_MTextBody mx_EventTile_content" } dir="auto">
```

Comment:
```suggestion
                    <span className="mx_MTextBody mx_EventTile_content" dir="auto">
```

Reply:
I think I removed this by intention because it broke rtl (either Chromium, Firefox or both) on one occasion. I've added `dir="auto"` on another place therefore and this should work in both. Actually I've tested the rtl stuff thoroughly and it worked better with the bubble layout and the normal layout.

Reply:
Oops, ignore, weird line break on mobile, didn't see that's not about `dir="auto"`. 🤪